drivers/backup/portworx/portworx.go

```diff
@@ -1040,6 +1061,31 @@ func (p *portworx) UpdateOwnershipRule(ctx context.Context, req *api.RuleOwnersh
 	return p.ruleManager.UpdateOwnership(ctx, req)
 }
 
+func (p *portworx) GetBackupUID(ctx context.Context, orgID, backupName string) (string, error) {
+	var backupUID string
+	var totalBackups int
+	bkpEnumerateReq := &api.BackupEnumerateRequest{OrgId: orgID}
+	bkpEnumerateReq.EnumerateOptions = &api.EnumerateOptions{MaxObjects: uint64(enumerateBatchSize), ObjectIndex: 0}
+	for {
+		enumerateRsp, err := p.EnumerateBackup(ctx, bkpEnumerateReq)
+		if err != nil {
+			return backupUID, fmt.Errorf("Failed to enumerate backups for org %s ctx: [%v]", orgID, err)
```

Comment:
```suggestion
			return "", fmt.Errorf("Failed to enumerate backups for org %s ctx: [%v]", orgID, err)
```

Reply:
The default string value declared with var is anyway "".